metricbeat/module/apache/_meta/config.yml

```diff
@@ -1,5 +1,4 @@
 - module: apache
-  metricsets: ["status"]
   period: 10s
 
   # Apache hosts
```

Comment:
I think we could drop the "Apache hosts" comment. It really doesn't any new info IMO.

Reply:
host comment removed and also period removed.